### ISSUE TITLE
Simple Test Error (tutorial section 4.2)

### ISSUE THREAD

**Issue by cherdeman** [dssg/dirtyduck#9]
When I run the command `./tutorial.sh triage --config_file simple_test_skeleton.yaml run` to run a simple experiment in section 4.2 of the tutorial I get the following error:

```
Traceback (most recent call last):
  File "/usr/local/bin/triage_experiment", line 11, in <module>
    load_entry_point('triage-experiment', 'console_scripts', 'triage_experiment')()
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1134, in invoke
    Command.invoke(self, ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/triage_experiment/triage_experiment.py", line 63, in triage
    replace=replace
  File "/usr/local/lib/python3.6/site-packages/triage/experiments/base.py", line 99, in __init__
    self.initialize_components()
  File "/usr/local/lib/python3.6/site-packages/triage/experiments/base.py", line 228, in initialize_components
    evaluator_config=self.config.get('scoring', {})
  File "/usr/local/lib/python3.6/site-packages/triage/component/catwalk/model_testers.py", line 28, in __init__
    metric_groups=evaluator_config['metric_groups'],
KeyError: 'metric_groups'
```
This looks like a triage version issue similar to #8, but I made the versioning fix locally that addressed that issue.

**Comment by ecsalomon** [comment]
Looks like an old version of triage is asking for `metric_groups`, which has been replaced in new triage (and in the config file) with `training_metric_groups` and `test_metric_groups`

**Comment by nanounanue** [comment]
True, fixed with 9c8de4bd9fd5cf907bfa80f96cd9240fd47cfdb8
